src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/pca/HDF5PCACoveragePoNCreationUtils.java

```diff
@@ -209,8 +209,8 @@ private static ReadCountCollection cleanNormalizedCounts(final ReadCountCollecti
         // Remove column and targets with too many zeros and targets with extreme median coverage:
         final int maximumColumnZerosCount = calculateMaximumZerosCount(readCounts.targets().size(), maximumPercentageZeroColumns);
         final int maximumTargetZerosCount = calculateMaximumZerosCount(readCounts.columnNames().size(), maximumPercentageZeroTargets);
-        ReadCountCollection cleanedCounts = ReadCountCollectionUtils.removeColumnsWithTooManyZeros(readCounts, maximumColumnZerosCount, true, logger);
-        cleanedCounts = ReadCountCollectionUtils.removeTargetsWithTooManyZeros(cleanedCounts, maximumTargetZerosCount, true, logger);
+        ReadCountCollection cleanedCounts = ReadCountCollectionUtils.removeColumnsWithTooManyZeros(readCounts, maximumColumnZerosCount, false, logger);
+        cleanedCounts = ReadCountCollectionUtils.removeTargetsWithTooManyZeros(cleanedCounts, maximumTargetZerosCount, false, logger);
```

Comment:
I'd be satisfied (for now) with a direct unit test for `cleanNormalizedCounts()` that checks that rounding is not performed on the return value. If "not rounding" is a post-condition for this method and part of the method's new contract, then there should be a direct unit test for that post-condition, even if this will later be supplemented by an end-to-end test for the affected tool(s). This could be done by making the method `@VisibleForTesting`.

The method documentation should also be updated to state that rounding is not performed.